src/engine/init.c

```diff
@@ -640,10 +640,10 @@ server_init(int argc, char *argv[])
 	D_INFO("Service fully up\n");
 
 	/** Report timestamp when engine was open for business */
-	d_tm_record_timestamp(NULL, "servicing_at", NULL);
+	d_tm_record_timestamp(NULL, "servicing_at");
 
 	/** Report rank */
-	d_tm_set_gauge(NULL, dss_self_rank(), "rank", NULL);
+	d_tm_set_gauge(NULL, dss_self_rank(), "rank");
```

Comment:
Shall we move this to a counter then with an initial value equal to the rank? We don't care about min/max/avg for this one.

Reply:
I will change this over, and push an update to obtain a green run through CI.